setup.php

```diff
@@ -133,43 +133,71 @@ function recursive_writable_directory_test($dir)
         </style>
     </head>
     <body>
-        <div class = 'mt-4 container'>
-            <div class="row">
-                <div class="row">
-                <div class="col-sm-12">
-                    <div class="mb-3 border-bottom">
-                        <h2>OpenEMR Setup <a class="oe-pull-away oe-help-redirect" data-target="#myModal" data-toggle="modal" href="#" id="help-href" name="help-href" style="color:#676666" title="Click to view Help"><i class="fa fa-question-circle" aria-hidden="true"></i></a></h2>
-                    </div>
-                </div>
-            </div>
+        <nav class="navbar navbar-expand navbar-light bg-light">
+            <a class="navbar-brand" href="#">OpenEMR Setup</a>
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
```

Comment:
Replace the default id from the BS4 doc `navbarSupportedContent` with something else